gravitee-apim-console-webui/src/portal/customization/catalog/category-list/category-list.component.html

```diff
@@ -0,0 +1,145 @@
+
+      <!-- Display Description Column -->
+      <ng-container matColumnDef="description">
+        <th mat-header-cell *matHeaderCellDef id="description">Description</th>
+        <td mat-cell *matCellDef="let element">{{ element.description }}</td>
+      </ng-container>
+
+      <!-- Display API Count Column -->
+      <ng-container matColumnDef="count">
+        <th mat-header-cell *matHeaderCellDef id="api-count">API Count</th>
+        <td mat-cell *matCellDef="let element">
+          {{ element.totalApis }}
+        </td>
+      </ng-container>
+
+      <!-- Display Actions Column -->
+      <ng-container matColumnDef="actions">
+        <th mat-header-cell *matHeaderCellDef id="actions"></th>
+        <td mat-cell *matCellDef="let element; let i = index">
+          <div class="categories__list__actions">
+            @if (element.hidden) {
+              <button
+                mat-button
+                (click)="showCategory(element)"
+                matTooltip="Show Category"
+                *gioPermission="{ anyOf: ['environment-category-u'] }"
+              >
+                <mat-icon svgIcon="gio:cloud-published"></mat-icon>
+              </button>
+            } @else {
+              <button
+                mat-button
+                (click)="hideCategory(element)"
+                matTooltip="Hide Category"
+                *gioPermission="{ anyOf: ['environment-category-u'] }"
+              >
+                <mat-icon svgIcon="gio:cloud-unpublished"></mat-icon>
+              </button>
+            }
+            <button
+              mat-button
+              aria-label="Button to edit category"
+              [routerLink]="['category', element.id]"
+              matTooltip="Edit"
+              *gioPermission="{ anyOf: ['environment-category-u'] }"
+            >
+              <mat-icon data-testid="api_list_edit_button">edit</mat-icon>
+            </button>
+            <button mat-button *gioPermission="{ anyOf: ['environment-category-d'] }" (click)="deleteCategory(element)" matTooltip="Delete">
+              <mat-icon svgIcon="gio:trash"></mat-icon>
+            </button>
+          </div>
+        </td>
+      </ng-container>
+
+      <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
+      <tr
+        mat-row
+        *matRowDef="let row; columns: displayedColumns"
+        data-testid="link_table_row"
+        cdkDrag
+        [cdkDragData]="row"
+        cdkDragLockAxis="y"
+      ></tr>
+      <!-- Row shown when there is no data -->
+      <tr class="mat-mdc-row mdc-data-table__row" *matNoDataRow>
+        <td class="mat-mdc-cell mdc-data-table__cell" [attr.colspan]="displayedColumns.length">
+          {{ 'There are no categories for this environment.' }}
+        </td>
+      </tr>
+    </table>
+  </mat-card-content>
+</mat-card>
+
+<gio-save-bar [form]="portalSettingsForm" [formInitialValues]="portalSettingsInitialValue" (submitted)="saveSettings()"></gio-save-bar>
```

Comment:
Since the reordering of categories is saved instantaneously, the save bar is not necessary.

Reply:
Removed